src/CoreVideo/CVDisplayLink.cs

```diff
@@ -57,6 +57,54 @@ internal CVDisplayLink (IntPtr handle, bool owns)
 			this.handle = handle;
 		}
 
+		[Mac (12,0)]
+		[DllImport (Constants.CoreVideoLibrary)]
+		static extern int CVDisplayLinkCreateWithCGDisplay (uint displayId, out IntPtr displayLink);
+
+		[Mac (12,0)]
+		public static CVDisplayLink FromDisplayId (uint displayId)
+		{
+			var result = CVDisplayLinkCreateWithCGDisplay (displayId, out IntPtr handle);
+			if (result != 0)
+				throw new Exception ($"Could not create display link for display {displayId}.");
```

Comment:
include the error code in the string ?
otherwise return `null`...
an exception without the error code seems unhelpful :)

Reply:
I prefer to return null and change it to CVDisplayLink? since the compiler is smarter.

Reply:
It does hide the error code... not sure if/when it's helpful

You could also have an `out int error` on the managed API and developers can pass `out var _` if they have no interest in the error code

Reply:
I did that with other of the C APIS, add an overload with the out error, in case it is useful, will do the same.